lib/outcome.rb

```diff
@@ -0,0 +1,18 @@
+class Outcome
+
+  def is_tie?(all_move_combos, full_board_bool)
+    full_board_bool && !winner?(all_move_combos) ? true : false
```

Comment:
You don't need a ternary operator for this! Also, I'd reconsider `full_board_bool` (avoid mentioning data types in method and variable names). Since the value of `full_board_bool` is a boolean, a better name would be `is_board_full` or `is_full_board`.

Reply:
renamed to is_board_full